packages/main/src/webComponents/Popover/Popover.stories.mdx

```diff
@@ -112,6 +112,37 @@ const MyComponentWithPopover = () => {
 };
 ```
 
+**Opening a `Popover` by reference and not by `id`**
+
+The web component exposes a way to pass a reference of an element instead of an `id` to the `opener` prop. Since this is not supported when passing the reference in a declarative way to a React `prop`, you have to attach the ref directly on the web component.
```

Comment:
```suggestion
This web component exposes a way to pass a reference of an element instead of an `id` to the `opener` prop. Since this is not supported when passing the reference in a declarative way to a React `prop`, you have to attach the ref directly on the web component.
```